### PR TITLE
LPS-154378 | master

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/LPS-154378
Submit for master.

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
@liuxingye The used macros have been pulled under BChan's [commit](https://github.com/brianchandotcom/liferay-portal/pull/118172/commits/819345af223785db59968a15d71b2486a5ec9292), waiting for the update

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/messageboards/MBThread.testcase [hunk 1]
```diff
@@ -690,12 +690,54 @@ definition {
...
+			threadBodyImageEdit = "DM Document Title");
+
+		PortletEntry.publish();
+
+		JSONFolder.addFolder(
+			dmFolderDescription = "DM Folder Description",
+			dmFolderName = "DM Folder Name",
+			groupName = "Guest");
+
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
```
Comment (liuxingye) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/messageboards/MBThread.testcase]: @jaydawu we can move file to folder here using JSON

Comment (jaydawu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/messageboards/MBThread.testcase]: got it, thank you

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by liuxingye** [comment]
Pull request submitted to https://github.com/austinchiang/liferay-portal/pull/1341.

**Pull request closed without merging**